src/components/Cart/Cart.jsx

```diff
@@ -42,14 +42,21 @@ export const Cart = () => {
                 Tax included. Free shipping applied to all orders.
               </p>
               <div className='flex items-start flex-col'>
-                <Button className='w-full mb-3' color='primary'>
+                <Button
+                  className='w-full mb-3'
+                  color='primary'
+                  onClick={(e) => navigate('/checkout')}
```

Comment:
You can use `Link` component from `react-router-dom` and use like this

```javascript
 <Button
        className='w-full mb-3'
         color='primary'
        as={Link}
        to="/checkout"
   >
        Checkout
    </Button>
    ```
    
    By doing so you don't need to use `useNavigate`. You should apply the same way for `Continue Shopping`.